Kevinf sprint9

kfansler opened the pull request:
Changes:
* OAuth section modified to include optional plain text (as opposed to using a hash). Not yet implemented for SDKs, so just the table and the cURL examples include the new simplified method.
* Query param for GET /sheets for ?rowsModifiedSince

kfansler committed:
sprint 9 changes

Dev Program link
rowsModifiedSince

kfansler committed:
links to 10000ft and Bridge

kfansler committed:
Partial rework of oauth section

Updated image
Added client_secret value to table.
Still need to finish combing through doc for issues.
Still need to fix sample code with an either/or type of thing for hash vs client_secret.

kfansler committed:
Oauth revision finished

abryan-smartsheet commented on source/_introduction.html.md.erb:
````diff
@@ -1116,7 +1128,9 @@ smartsheet.tokens.refreshAccessToken(options)
 ```csharp
-// [todo]
+Token tokenRefreshed = oauth.RefreshToken(
+  token           // Token token
````
This comment doesn't really add anything helpful, IMO. Can this be condensed into:
Token tokenRefreshed = oauth.RefreshToken(token);

kfansler replied:
Agreed.

abryan-smartsheet commented on source/_introduction.html.md.erb:
````diff
@@ -1116,7 +1128,9 @@ smartsheet.tokens.refreshAccessToken(options)
+);
 ```
 
 ```java
````
Token refreshToken = oAuthFlow.refreshToken(token);

kfansler committed:
Removed unnecessary comment

Token token isn't useful

kfansler committed:
Stubs for plain text oauth

Make it clear that the SDKs do not yet support plain text for Oauth

kfansler committed:
Making multi-picklist show in search

Make it's own second-level topic mostly repurposed from the versioning topic.

kfansler committed:
ESC-1907

Added information about system userIds to Event Reporting

PauloLisboa commented on source/_enterprise.html.md.erb:
```diff
@@ -4,6 +4,21 @@ The following section describes the specific event types and payloads returned f
 
 With Event Reporting, you can programmatically retrieve events that are occurring in your Smartsheet organization account. Examples of events are create, update, load, and delete of sheets, reports, dashboards, attachments, and users. Note: while the current list of events covers most of Smartsheet functionality, new events will be added to the list as Smartsheet introduces new functionality.
 
+Some actions in Smartsheet are performed by system components. Therefore, the userId property in some response payloads refers to a system userId and not to a regular user. For example, the Smartsheet Events API shows 5629504736520068 as the userId when an anonymous user accesses a sheet that is published for anyone. 
```
Minor suggestion (just changed from`system userId` to `system user`):
the userId property in some response payloads refers to a system **user** and not to a regular user

kfansler replied:
Thanks!

kfansler committed:
change userId to user

kfansler committed:
Added filters detail for search

Added "object model" to header for search optimization
Added filtering section for search
Added functions stub for search

kfansler committed:
Changes for Ian and Kara

Formulas section
Fixed OAuth per Ian's review

kfansler committed:
add changelog

kfansler committed:
Update _reference-g-s.html.md.erb

small date change tip

kfansler committed:
admin rule

kfansler committed:
Clarification on Oauth change

kfansler committed:
less editorializing

kfansler committed:
added include=profileImage

Get Contact

kfansler committed:
include param
